test/tests/time_steppers/logconstant_dt/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [./logconstant_dt]
+    type = 'Exodiff'
+    input = 'logconstant_dt.i'
+    exodiff = 'logconstant_dt_out.e'
```

Comment:
I used the same test than for the other time steppers actually, to remain consistent between the different tests. Should I still go for a CSV output?